### ISSUE TITLE
bug: 

### ISSUE THREAD

**Issue by Lehvant** [uxbox/uxbox#2543]
### Steps To Reproduce

Steps to how I reproduced this behavior:
1. Go to 'Assets' tab on the left side nav
2. Add a color asset
3. Double click the name and try to place the caret in other areas within the text.
4. Also try selecting certain words.

As a workaround I've been using the left and right keys to move the caret around. 

### Expected behavior

The mouse caret should move to where the user clicks within the text. 

### Actual behavior

The caret doesn't move around where the user click within the text.

### Screenshots or video

https://user-images.githubusercontent.com/48232326/199577270-0a2186aa-b43c-4bec-be9a-5adbeab34022.mov

### Desktop (please complete the following information)

-Mac OS Ventura 13.0

### Environment (please complete the following information)

design.penpot.app

### Additional context

Thanks for all the hard work put into Penpot!

**Comment by myfunnyandy** [comment]
hi @abeerkhalaf, we could not reproduce the error. Can you check if its still happening?

**Comment by niekvb** [comment]
This is also happening on Windows 10 using Firefox 106.0.5.

This is also a problem when trying to edit the name of a page or page element (in tree)

**Comment by niekvb** [comment]
Probably related to this bug: https://bugzilla.mozilla.org/show_bug.cgi?id=800050

***TLDR***: Attribute `draggable` causes issues in `<li class="selected" draggable="true">`.

**Comment by niekvb** [comment]
@myfunnyandy Is this issue being tracked?

@abeerkhalaf Could you verify that you experienced this issue when using Firefox? Also, could you update the issue's title to be more descriptive?

**Comment by myfunnyandy** [comment]
@niekvb sorry I've missed this one. I've just created the issue in our tracker so we can take a deeper look: https://tree.taiga.io/project/penpot/issue/4572
